fluent-package/apt/systemd-test/update-from-v4.sh

```diff
@@ -49,6 +50,9 @@ esac
 systemctl status --no-pager fluentd
 systemctl status --no-pager td-agent
 
+# Fluentd should be restarted when update from v4.
+test $main_pid -ne $(eval $(systemctl show fluentd --property=MainPID) && echo $MainPID)
```

Comment:
Thanks for this test enhancement.

For some reason, the restart doesn't happen in deb.

Only when updating from v4 to v5, the behavior seems different.
(Normally, in deb, restart happens when the service was active before the update.)

I don't think it's as expected, but updating from v4 to v5 would normally be done after stopping the service, so this behavior would not be a significant issue.

In the meantime, please reverse this expectation and leave a comment.